packages/lb-utils/src/types/pointCloud.ts

```diff
@@ -89,6 +89,23 @@ export interface IPointCloudBox extends IVolume {
   newPointList?: ICoordinate[];
 }
 
+export interface IPointCloudSphere {
+  attribute: string;
+  center: I3DSpaceCoord;
+  radius: number,
+  widthSegments: number,
+  heightSegments: number,
+  id: string;
+  valid: boolean;
+  trackID?: number;
+  subAttribute?: {
+    [k: string]: string;
+  };
+  count?: number;
```

Comment:
widthSegments 、 heightSegments、count、radius  - Delete it.

center => position.